### PR TITLE
Opfab cli : rename commands for log (#7337)

### LINKED ISSUE #7337: Opfab cli : rename commands for log
Instead of :  

`opfab service get-log-level <serviceName> `

do  

`opfab log get-level  <serviceName>`

Instead of :  

`opfab service set-log-level <serviceName> <level>`

do  

`opfab log set-level  <serviceName> <level>`

### PR DESCRIPTION
**Pull request by vlo-rte**
- In release notes :
  -  In chapter : Features
  -  Text : #7337 : Opfab cli : rename commands for log

### CODE REVIEW

File: cli/src/commands.js [hunk 1]
```diff
@@ -18,7 +18,7 @@ const perimeters = require('./perimetersCommands.js');
 const processGroups = require('./processGroupsCommands.js');
 const realtimescreen = require('./realtimescreenCommands.js');
 const businessData = require('./businessDataCommands.js');
-const service = require('./serviceCommands.js');
+const service = require('./logCommands.js');
```
Comment (quinarygio) [inline cli/src/commands.js]: Rename "const service" to "const log"

File: cli/src/commands.js
Comment (quinarygio) [inline cli/src/commands.js:1]: In printHelp function is still referenced the "service" command (row 204 and 246) :

        service                 Get or set log level for services

...

                case 'service':
                    service.printHelp();
                    break;

**Review by quinarygio — CHANGES_REQUESTED** [review]

File: cli/src/commands.js (hunk 1 above)
Comment (vlo-rte) [reply cli/src/commands.js]: Done.

File: cli/src/commands.js
Comment (vlo-rte) [reply cli/src/commands.js:1]: Done.

**Review by quinarygio — APPROVED** [review]

**Pull request merged**